iron-list.html

```diff
@@ -209,6 +216,33 @@
       },
 
       /**
+       * When true, `iron-list` will render the items in a fixed-width grid.
+       * If true the `itemWidth` property will also need to be set
+       */
+      grid: {
+        type: Boolean,
+        value: false

```

Comment:
it needs `reflectToAttribute: true` because you can toggle the `grid` property programmatically and you are styling against it in line 141.